server/lib/pbench/__init__.py

```diff
@@ -153,7 +153,8 @@ def get_pbench_logger(caller, config):
         except FileExistsError:
             # directory already exists, ignore
             pass
-        fh = logging.FileHandler(os.path.join(logdir, "{}.log".format(caller)))
+        #fh = logging.FileHandler(os.path.join(logdir, "{}.log".format(caller)))
+        fh = SysLogHandler(address=('localhost', 514), facility=19, socktype = socket.SOCK_STREAM)
```

Comment:
Great!

Can you also verify that just using `/dev/log` also works?  See ["How to Configure Logging to Syslog in Python"](https://stackoverflow.com/questions/3968669/how-to-configure-logging-to-syslog-in-python).

And then we need to make the use of the `FileHandler` and `SyslogHandler` configurable so that for the unit test environment we can use `FileHandler` and all other environments use `SyslogHandler`.

Reply:
@portante made the 1st change i.e. localhost to /dev/log . Will work on second part